src/Sulu/Bundle/AdminBundle/Resources/js/views/List/itemActions/ResourceDetailLinkItemAction.js

```diff
@@ -0,0 +1,69 @@
+// @flow
+import AbstractListItemAction from './AbstractListItemAction';
+
+export default class ResourceDetailLinkItemAction extends AbstractListItemAction {
```

Comment:
Maybe just name it `DetailLinkItemAction` ? Would be a bit shorter 🤔

Reply:
Okay then we name it also `detail_link` in the PHP Code?

Reply:
yes I think it is a bit shorter and cleaner, and we don't lose important information